stories/global.scss

```diff
@@ -66,3 +66,9 @@ table {
   position: inherit!important;
 }
 
+// StringList.stories.mdx
+.string-list-footer {
+  button {
+    margin-right: 0px;
+  }
+}
```

Comment:
This class override is required because we have a css class on top of this file that applies to every buttons in the Storybook:

```
// Ensure buttons have spacing between them
button {
  margin-right: 10px;
}
```
wouldn't it be better not to use a generic class for each story?

Reply:
https://github.com/rancher/dashboard/issues/7407
As discussed we'll handled in another issue.